src/components/common/layout/Section.js

```diff
@@ -60,10 +61,7 @@ const styles = StyleSheet.create({
   section: {
     backgroundColor: '#ffffff',
     borderRadius: normalize(5),
-    padding: normalize(10),
-    paddingTop: normalize(15),
-    paddingBottom: normalize(15),
-    marginBottom: normalize(15)
+    padding: normalize(8)
```

Comment:
this should come from the theme I think. I've call it in my branch defaultMargin, perhaps we can use the same or a defaultPadding

Same for border radius.